packages/story-editor/src/components/library/panes/shopping/productList.js

```diff
@@ -0,0 +1,224 @@
+  ${imgPlaceholder}
+`;
+
+const StyledImage = styled.img`
+  ${imgPlaceholder}
+`;
+
+const StyledDescription = styled.div`
+  padding-left: 12px;
+`;
+
+const StyledTitle = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM,
+})`
+  color: ${({ theme }) => theme.colors.fg.primary};
+`;
+
+const StyledPrice = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+`;
+
+const onPageIcons = (
+  <>
+    <Icons.Checkmark className="check" /> <Icons.Cross className="remove" />
+  </>
+);
+function ProductList({
+  products = [],
+  onPageProducts = [],
+  onInsert,
+  onRemove,
+  isMenuFocused = true,
+}) {
+  const { handleListNav, currentRowsRef, setCurrentFocusIndex } =
+    useProductNavigation({ isMenuFocused, products });
+
+  useEffect(() => {
+    if (!isMenuFocused) {
+      setCurrentFocusIndex(0);
+    }
+  }, [isMenuFocused, setCurrentFocusIndex]);
+
+  return (
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions -- list handles arrow up and arrow down
+    <div role="list" onKeyDown={handleListNav}>
+      {products.map((product, index) => {
+        const imageSrc = product?.productImages[0]?.url || '';
+        const onPage = isOnPage(product, onPageProducts);
+        const handleClick =
+          onPage && onPage?.productId
+            ? () => {
+                onRemove && onRemove(product);
+              }
+            : () => {
+                onInsert && onInsert(product);
+              };
+
+        const handleFocus = (evt) => {
+          if (currentRowsRef.current[`row-${index}`] && evt.type === 'focus') {
+            setCurrentFocusIndex(index);
+          }
+        };
+
+        const ADD_PRODUCT_TEXT = sprintf(
+          /* translators: %s: add product title.*/
+          __('Add %s', 'web-stories'),
+          product.productTitle
+        );
+
+        const REMOVE_PRODUCT_TEXT = sprintf(
+          /* translators: %s: remove product title.*/
+          __('Remove %s', 'web-stories'),
+          product.productTitle
+        );
+
+        return (
+          <StyledListItem
+            id={`row-${index}`}
+            role="listitem"
+            key={product?.productTitle}
+            tabIndex={-1}
+            ref={
+              (el) => (currentRowsRef.current[`row-${index}`] = el) // track the active row
+            }
+          >
+            <StyledActionsContainer>
+              <Button
+                aria-label={onPage ? REMOVE_PRODUCT_TEXT : ADD_PRODUCT_TEXT}
+                onClick={handleClick}
+                type={BUTTON_TYPES.TERTIARY}
+                size={BUTTON_SIZES.SMALL}
+                variant={BUTTON_VARIANTS.SQUARE}
+                onFocus={handleFocus}
+              >
+                {onPage ? onPageIcons : <Icons.PlusFilled />}
+              </Button>
+            </StyledActionsContainer>
+            {imageSrc ? (
+              <StyledImage alt={product?.productTitle} src={imageSrc} />
```

Comment:
Need to add the `crossOrigin="anonymous"` attribute here, otherwise images do not load when video optimization (and thus cross-origin isolation) is turned on.

```suggestion
              <StyledImage
                alt={product?.productTitle}
                src={imageSrc}
                crossOrigin="anonymous"
              />
```

Reply:
Now using

```
<StyledImage
      alt={product?.productTitle}
      src={imageSrc}
      loading="lazy"
      decoding="async"
      crossOrigin="anonymous"
    />
```